providers/src/airflow/providers/celery/provider.yaml

```diff
@@ -329,6 +329,13 @@ config:
         type: string
         example: ~
         default: "False"
+      extra_celery_config:
+        description: |
+          Extra celery configs to include in the celery worker
```

Comment:
Can you add a reference to celery page where possible parameters are documented?

Reply:
Added the reference to all the configurations which can be applied using this airflow helm config as well as added a sample config also for better understanding.